api/RTCIceTransport.json

```diff
@@ -741,56 +691,6 @@
           }
         }
       },
-      "selectedcandidatepairchange_event": {
```

Comment:
https://github.com/mdn/browser-compat-data/blob/820c07582578aab9b125ef8923d3b11f3f108cc1/api/RTCIceTransport.json#L597-L645

Reply:
I wonder there should be a way to do the sync automatically, or at least a linter for this 🤔